input/fsh/who-co-Observations.fsh

```diff
@@ -0,0 +1,294 @@
+Parent: Observation
+Id: who-co-observation-outcome-patcourse-vent
+Title: "WHO CO Outcome Patcourse Vent"
+Description: "WHO CO Outcome Patcourse Vent Observation"
+* code = $LNC#96539-2
+* value[x] only CodeableConcept
+* valueCodeableConcept 1..1
+* valueCodeableConcept from WhoCrValueSetYesNoUnk
+
+Profile: WhoCoObservationOutcomePatcourseEcmo
+Parent: Observation
+Id: who-co-observation-outcome-patcourse-ecmo
+Title: "WHO CO Outcome Patcourse Ecmo"
+Description: "WHO CO Outcome Patcourse Ecmo Observation"
+* code = $LNC#96540-0
+* value[x] only CodeableConcept
+* valueCodeableConcept 1..1
+* valueCodeableConcept from WhoCrValueSetYesNoUnk
+
+Profile: WhoCoObservationOutcomePatcourseStatus
+Parent: Observation
+Id: who-co-observation-outcome-patcourse-status
+Title: "WHO CO Outcome Patcourse Status"
+Description: "WHO CO Outcome Patcourse Status Observation"
+* code = $LNC#91541-3
+* value[x] only CodeableConcept
+* valueCodeableConcept 1..1
+* valueCodeableConcept from WhoCoValueSetHealthOutcome
+
+Profile: WhoCoObservationOutcomePatcourseStatusOther
+Parent: Observation
+Id: who-co-observation-outcome-patcourse-status-other
+Title: "WHO CO Outcome Patcourse Status Other"
+Description: "WHO CO Outcome Patcourse Status Other Observation"
+* value[x] only string
+* valueString 0..1
+
+Profile: WhoCoObservationOutcomeLabDate
+Parent: Observation
+Id: who-co-observation-outcome-lab-date
+Title: "WHO CO Outcome Lab Date"
+Description: "WHO CO Outcome Lab Date Observation"
+* code = $LNC#96550-9
+* value[x] only dateTime
+* valueDateTime 0..1
+
+Profile: WhoCoObservationOutcomeLabResult
+Parent: Observation
+Id: who-co-observation-outcome-lab-result
+Title: "WHO CO Outcome Lab Result"
+Description: "WHO CO Outcome Lab Result Observation"
+* code = $LNC#96552-5
+* value[x] only CodeableConcept
+* valueCodeableConcept 0..1
+* valueCodeableConcept from WhoCoValueSetPositiveNegativeUnknown
+
+Profile: WhoCoObservationOutcomeContactsFollowed
+Parent: Observation
+Id: who-co-observation-outcome-contacts-followed
+Title: "WHO CO Outcome Contacts Followed"
+Description: "WHO CO Outcome Contacts Followed Observation"
+* code = $LNC#96551-7
+* value[x] only integer
+* valueInteger 1..1
+
+Profile: WhoCoObservationOutcomeContactsFollowedUnknown
+Parent: Observation
+Id: who-co-observation-outcome-contacts-followed-unknown
+Title: "WHO CO Outcome Contacts Followed Unknown"
+Description: "WHO CO Outcome Contacts Followed Unknown Observation"
+* code = $LNC#96552-5
+* value[x] only CodeableConcept
+* valueCodeableConcept 0..1
+* valueCodeableConcept from WhoCrValueSetYesNoUnk
+
+/*
+ * Examples 
+ */
+
+Instance: WhoCoObservationOutcomeAsympExample
+InstanceOf: WhoCoObservationOutcomeAsymp
+Usage: #example
+Title: "WHO CO Outcome Asymp Example"
+Description: "Developed symptoms after time of specimen collection"
+* status = #final
+* valueCodeableConcept = $cs-v2-0136#Y
+
+Instance: WhoCoObservationOutcomeAsympDateExample
```

Comment:
Please add this date value to the Observation above as an `effectiveDateTIme`.
Maybe you could also include an Observation in the negative (which would then exclude the date).
We won't necessarily use it in the Bundle but it'll be good to show the alternative form of the Observation.